Makefile

```diff
@@ -1,7 +1,10 @@
 QDL := qdl
 RAMDUMP := qdl-ramdump
 
-CFLAGS += -O2 -Wall -g `pkg-config --cflags libxml-2.0 libusb-1.0`
+BUILD_TIME := $(shell date +"%c")
```

Comment:
This makes the build unreproducible. Should use SOURCE_DATE_EPOCH, see https://reproducible-builds.org/docs/source-date-epoch/

Reply:
Why is this information valuable. Please use the commit message to describe your motivation.

Reply:
sure, as soon as I will debug non working qdl at the PC of my colleague may be I will return to this change ;-)